src/Engine/Scene/GeometryComponent.hpp

```diff
@@ -98,10 +100,11 @@ class SurfaceMeshComponent : public GeometryComponent
     std::shared_ptr<RenderMeshType> m_displayMesh { nullptr };
 };
 
-using TriangleMeshComponent = SurfaceMeshComponent<Ra::Core::Geometry::TriangleMesh>;
-using LineMeshComponent     = SurfaceMeshComponent<Ra::Core::Geometry::LineMesh>;
-using QuadMeshComponent     = SurfaceMeshComponent<Ra::Core::Geometry::QuadMesh>;
-using PolyMeshComponent     = SurfaceMeshComponent<Ra::Core::Geometry::PolyMesh>;
+using TriangleMeshComponent        = SurfaceMeshComponent<Ra::Core::Geometry::MultiIndexedGeometry>;
+using GeometryDisplayableComponent = SurfaceMeshComponent<Ra::Core::Geometry::MultiIndexedGeometry>;
+using LineMeshComponent            = SurfaceMeshComponent<Ra::Core::Geometry::LineMesh>;
```

Comment:
Why is this one not a `SurfaceMeshComponent<Ra::Core::Geometry::MultiIndexedGeometry>` also ?

Reply:
I removed this delcaration, seems that it is not used.